lib/watcher/status-wd.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
```

Comment:
I don't need this. I need the worker template. The worker template looks at all the files in the watcher folder and expects worker() method. If I add return in the beginning of the file, the worker method is no longer available for worker.js line 59 and we will face an error ```watcher.worker is not a function```